src/pkg/notifier/model/event.go

```diff
@@ -42,6 +42,7 @@ type EventData struct {
 	Repository  *Repository        `json:"repository,omitempty"`
 	Replication *model.Replication `json:"replication,omitempty"`
 	Retention   *model.Retention   `json:"retention,omitempty"`
+	ScanType    string             `json:"scan_type,omitempty"`
```

Comment:
The `EventData` is a common data structure for the event, the content inside might need to be abstracted into new structures according to modules, for the sake of future expansion, should we abstract a `Scan` struct and put `ScanType` inside it? Just like replication and retention.

Reply:
Updated